string map_file
string config_file
---
bool Completed